src/storage/txn/commands/acquire_pessimistic_lock.rs

```diff
@@ -46,9 +46,9 @@ command! {
             /// later read in the same transaction.
             return_values: bool,
             min_commit_ts: TimeStamp,
-            old_values: OldValues,
```

Comment:
Why removed the field `old_values`? Do we still support this feature of `old_value`?
um..., I know little about `old_value`.

Reply:
It seems that the old_value is never assigned as any valid value outside the command, and its only purpose is to hold the results produced during the pessimistic lock command and return it after finishing the command, which seems can be done by a local variable.